tests/link/numba/test_blockwise.py

```diff
@@ -58,3 +58,15 @@ def test_blockwise_benchmark(benchmark):
     x_test = np.eye(3) * np.arange(1, 6)[:, None, None]
     fn(x_test)  # JIT compile
     benchmark(fn, x_test)
+
+
+def test_repeated_args():
+    x = tensor3("x")
+    x_test = np.full((1, 1, 1), 2.0, dtype=x.type.dtype)
+    out = x @ x
+    fn, _ = compare_numba_and_py([x], [out], [x_test], eval_obj_mode=False)
```

Comment:
I don't get it, where's the repeated arg?

Reply:
dot with x twice